code/modules/hippie/code/game/objects/effects/holy.dm

```diff
@@ -0,0 +1,24 @@
+//Effects used for anything holy and divine currently used for smite and spawming a admin dummy... maybe for holy explosion one day
+/obj/effect/holy
+	name = "holy"
+	icon = 'icons/hippie/effects/holy.dmi'
+	layer = MOB_LAYER+1
+	use_fade = 0 //This should really be FALSE/TRUE I think
```

Comment:
I would, but don't I have to follow whatever the parent is using? this was inherited from /obj/effect which is using 1 and 0 if I just changed this one child to use TRUE/FALSE won't it break something?